### DISCUSSION TITLE
SVG asset preview missing

### DISCUSSION THREAD

**Discussion by paulhaem** [pimcore/pimcore#14211] [Q&A / Support]
SVG assets are not showing any preview when viewed in the folder view or detail view. Instead a placeholder image is shown (in the folder view) or nothing at all (detail view):

![image](https://user-images.githubusercontent.com/6862218/216253054-6a9fc1af-fbcf-4475-a8bc-c859a0352aa7.png)

The system check shows that Imagick is available and thumbnails for SVGs are properly generated:

![image](https://user-images.githubusercontent.com/6862218/216253183-76d96478-10f3-4dd6-bfa5-7e07b78e32ba.png)

Any idea what could cause this?

**Comment by blankse** [comment]
So your imagemagick doesn't support SVG. When you are on debian you can try `apt-get update && apt-get install libmagickcore-dev` see https://stackoverflow.com/questions/59370356/how-to-enable-svg-support-for-imagemagick

**Reply by paulhaem** [reply]
Thanks for looking into this! Inside the local docker container I use for development `libmagickcore-dev` is also missing but the previews are generated. Fun thing is that SVG thumbnails inside Twig templates are generated fine but only the JPG preview in the admin interface is not being generated.

**Reply by paulhaem** [reply]
When generating the thumbnails for a SVG using the Pimcore command it generates "filetype-not-supported.jpg" for the treeview thumbnail. The other thumbnails for the SVG are generated fine.
